apps/qubit/modules/default/actions/moveAction.class.php

```diff
@@ -111,11 +111,18 @@ public function execute($request)
       $page = $request->page;
     }
 
-    // Avoid pagination over 10000 records
-    if ((int)$limit * $page > 100)
+    // Avoid pagination over ES' max result window config (default: 10000)
+    $maxResultWindow = arElasticSearchPluginConfiguration::getMaxResultWindow();
+
+    if ((int)$limit * $page > $maxResultWindow)
```

Comment:
I'm not going to keep flagging the type cast operator without trailing space, but there are many more. :)